src/resources/views/base/inc/scripts.blade.php

```diff
@@ -1,9 +1,20 @@
 @if (config('backpack.base.scripts') && count(config('backpack.base.scripts')))
     @foreach (config('backpack.base.scripts') as $path)
-    <script type="text/javascript" src="{{ asset($path).'?v='.config('backpack.base.cachebusting_string') }}"></script>
+        @if(is_array($path))
+            @basset(...$path)
+        @else
+            @basset($path)
+        @endif
     @endforeach
 @endif
 
+@basset('https://cdnjs.cloudflare.com/ajax/libs/pace/1.2.4/pace.min.js')
+@basset('https://cdnjs.cloudflare.com/ajax/libs/noty/3.1.4/noty.min.js')
+@basset('https://cdnjs.cloudflare.com/ajax/libs/sweetalert/2.1.2/sweetalert.min.js')
+@basset('https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.3/jquery.min.js')
+@basset('https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.6.2/js/bootstrap.min.js')
+@basset('https://unpkg.com/@coreui/coreui@2.1.16/dist/js/coreui.js')
```

Comment:
Heads-up @promatik - this is important. What I've done here is that I've removed the `bundle.js` file from CRUD. It was just merging all of these together. So instead of needing that part of the build script... boom, the blade file is loading them. One less thing to build.